### ISSUE TITLE
Update colour of culture section text so it meets WCAG 2.1 AA colour contrast

### ISSUE THREAD

**Issue by danivazquez44** [guardian/newsletters-nx#393]
Current hex code of brown text colour should be #866D50, not #A1845C, to pass contrast ratio for accessibility.

<img width="1040" height="356" alt="Image" src="https://github.com/user-attachments/assets/68f0b10e-cce9-4618-9cc4-dd7c6cb00c31" />
